src/DirectiveParser.ump

```diff
@@ -19,40 +22,46 @@ class DirectiveParser {
     
     
     // JET files need to start with the directive. So if they don't will just ignore it
-    // I assume that the directive is contained on one line
+    // Assumes that the directive is contained on one line
```

Comment:
It is not always correct. I created issue #8 for this since I don't think I will have enough time.